src/style.css

```diff
@@ -2,0 +1,29 @@
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+}
+
+.stars__star {
+  margin: 0;
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(./images/star.svg);
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
```

Comment:
Consider using a consistent unit for margins. For example, if you're using `px` elsewhere, stick to it unless there's a specific reason to use a different unit.